crates/chainspec/src/spec.rs

```diff
@@ -258,6 +258,13 @@ impl From<ForkBaseFeeParams> for BaseFeeParamsKind {
     }
 }
 
+/// The type for specify which is kind of engine api
+#[derive(Debug)]
+pub enum EngineApiKind {
+    Ethereum,
```

Comment:
this should be default

Reply:
@mattsse, what does you means by defaut?